.github/std-unexploded.yml

```diff
@@ -184,7 +184,7 @@ jobs:
       name: dev-preview
       url: https://backend.dev-preview.eks.lw.iog.io
     # Boolean input should be compared with string until https://github.com/actions/runner/issues/2238 resolved
-    if: fromJSON(needs.discover.outputs.hits).deployments.apply != '{}' && github.event.pull_request.merged == 'true'
+    if: fromJSON(needs.discover.outputs.hits).deployments.apply != '{}' && (github.event.pull_request.merged == 'true' || github.ref == 'refs/heads/master')
```

Comment:
Can this not be done exclusively with a [push event](https://docs.github.com/en/rest/overview/github-event-types?apiVersion=2022-11-28#pushevent)?

Reply:
yes! 1 min

Reply:
No, wait. I'm not sure if a merge (of a PR) counts as push event.

Reply:
Addressed in [force-push](https://github.com/input-output-hk/cardano-js-sdk/compare/1347c1c444259e4dffe10b5a3e22907507108896..4f5614ebf37f1b5849c075b62b6b917ab944e3b2)

Reply:
Ah, no you're right: https://github.com/input-output-hk/cardano-js-sdk/blob/daa05d97a928ed1878bba16848ea40618979e50b/.github/workflows/continuous-integration-unit-tests.yaml#L7-L12

Will likely need to consider both events

Reply:
Hang on, that's actually triggering the workflow to run before merge. Let's just see if it runs after merging this PR